### PR TITLE
remove mobile app feature flag

### PR DESCRIPTION
**Pull request by joeyorlando**
# What this PR does

## Checklist

- [x] Tests updated
- [ ] Documentation added (N/A)
- [x] `CHANGELOG.md` updated

### CODE REVIEW

File: engine/engine/urls.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
     path("integrations/v1/", include("apps.integrations.urls", namespace="integrations")),
     path("twilioapp/", include("apps.twilioapp.urls")),
     path("api/v1/", include("apps.public_api.urls", namespace="api-public")),
+    path("mobile_app/v1/", include("apps.mobile_app.urls", namespace="mobile_app")),
+    path("api/internal/v1/mobile_app/", include("apps.mobile_app.urls", namespace="mobile_app_tmp")),
```
Comment (joeyorlando) [inline engine/engine/urls.py:39]: I'm wondering if `api/internal/v1/mobile_app/*` routes are actually used?

File: grafana-plugin/src/containers/UserSettings/parts/connectors/MobileAppConnector.tsx [hunk 2]
```diff
@@ -4,30 +4,22 @@ import { Button, Label } from '@grafana/ui';
...
 
 const cx = cn.bind(styles);
 
-interface SlackConnectorProps {
+interface MobileAppConnectorProps {
   onTabChange: (tab: UserSettingsTab) => void;
 }
 
-const SlackConnector = (props: SlackConnectorProps) => {
+const MobileAppConnector = (props: MobileAppConnectorProps) => {
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/UserSettings/parts/connectors/MobileAppConnector.tsx:16]: rename from `SlackConnector` to `MobileAppConnector` (similarly w/ props interface above)

**Review by Matvey-Kuk — APPROVED** [review]

File: engine/engine/urls.py (hunk 1 above)
Comment (imtoori) [reply engine/engine/urls.py:39]: yes, we're using 
'api/internal/v1/mobile_app/auth_token'

Comment (imtoori) [reply engine/engine/urls.py:39]: and also 
'api/internal/v1/mobile_app/fcm'

Comment (joeyorlando) [reply engine/engine/urls.py:39]: thanks. I’ll leave these untouched for now. if we want to remove a set, can do it in a future PR

**Review by vadimkerr — APPROVED** [review]

**Pull request merged**